### PR TITLE
Updates towards public preview release on March 1st

### PR DESCRIPTION
**Pull request by isvargasmsft**
Work item: https://tasks.office.com/microsoft.onmicrosoft.com/Home/Task/1OWjnxIwjk26uWLDJck1OJUAOP5R?Type=TaskLink&Channel=Link&CreatedTime=638443200372410000

### CODE REVIEW

File: docs/includes/preview-disclaimer.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ ms.topic: include
 <!-- markdownlint-disable MD041-->
 
 > [!IMPORTANT]
-> Plugins for Microsoft Copilot for Microsoft 365 are in preview and only work in Microsoft 365 Chat in Microsoft Teams. During the preview, production tenants must [opt-in to enable plugins](../prerequisites.md#requirements-for-plugin-development).
+> Plugins for Microsoft Copilot for Microsoft 365 are in public preview and only work in Microsoft 365 Chat in Microsoft Teams. The capability is enabled by default in all Microsoft 365 Copilot licensed tenants. Admins can disable this functionality on a user and group basis and control how individual plugins are approved for use, and which plugins are enabled.
```
Comment (erikadoyle) [inline docs/includes/preview-disclaimer.md:9]: Is there an admin docs article that we can link to?

File: docs/prerequisites.md [hunk 2]
```diff
@@ -9,26 +9,29 @@ ms.date: 1/17/2024
...
 
-This topic summarizes the developer environment prerequisites and requirements to extend, enrich, and customize Copilot for Microsoft 365 for the unique way your customers work.
+This article summarizes the developer environment prerequisites and requirements to extend, enrich, and customize Copilot for Microsoft 365 for the unique way your customers work.
 
 ## Prerequisites
 
-Developers have two ways to obtain a developer environment for Copilot: (1) with a sandbox Microsoft 365 tenant with Copilot (available in limited preview through [TAP membership](#isv-partners-in-microsoft-365-developer-tap)), or (2) with an eligible Microsoft 365 or Office 365 production environment with a Copilot for Microsoft 365 license.
+Developers have two ways to obtain a developer environment for Copilot:
+
+1. With a sandbox Microsoft 365 tenant with Copilot (available in limited preview through [TAP membership](#isv-partners-in-microsoft-365-developer-tap)).
```
Comment (erikadoyle) [inline docs/prerequisites.md:20]: Let's define the "TAP" acronym upon first usage of the article too, like you do with ISVs in the following text

File: docs/prerequisites.md [hunk 3]
```diff
@@ -44,24 +47,24 @@ To learn more and choose the best extensibility path for you and your customer,
...
+As a developer, you need the ability to [sideload custom apps](/microsoftteams/platform/concepts/deploy-and-publish/apps-upload) in order to install, test, and debug message extension apps and plugins directly in the Teams environment without having to publish to your organization. An administrator must [enable custom app uploading](/microsoftteams/teams-custom-app-policies-and-settings#allow-users-to-upload-custom-apps) in your Microsoft 365 tenant (sandbox or production).
 
 ### Enabling developer mode
 
-You can use *developer mode* for M365 Chat while testing your plugin to verify if and how the orchestrator selected your plugin for use in response to a given prompt. From  *M365 Chat*, you can enable developer mode by typing `-developer on` (or `off` to disable).
+You can use *developer mode* for Microsoft 365 Chat while testing your plugin to verify if and how the orchestrator selected your plugin for use in response to a given prompt. From  *M365 Chat*, you can enable developer mode by typing `-developer on` (or `off` to disable).
```
Comment (erikadoyle) [inline docs/prerequisites.md:63]: Did you mean to retain `*M365 Chat*` here?

File: docs/includes/preview-disclaimer.md (hunk 1 above)
Comment (erikadoyle) [reply docs/includes/preview-disclaimer.md:9]: How about this one: https://learn.microsoft.com/en-us/microsoft-365/admin/manage/manage-plugins-for-copilot-in-integrated-apps?view=o365-worldwide

Comment (erikadoyle) [reply docs/includes/preview-disclaimer.md:9]: Also, let's find answers for these questions that (dev) customers might have and document them here:

- Will previously and/or newly installed Teams MEs automatically start lighting up as Copilot plugins?
- If so, is there a way to opt-out my ME from being a plugin (e.g., if I'm not ready to support that experience)?
- If not, how can I opt-in my ME as a Copilot plugin? (Does my app need to be resubmitted / reinstalled?)

Comment (isvargasmsft) [reply docs/includes/preview-disclaimer.md]: @erikadoyle reply from Kalyan:
1. Teams ME, which have been converted to MoS, approved by store validation team as works in copilot (For LOB all installed MoS MEs), enabled by tenant admin will show up in the flyout on copilot. After all this user has to explicitly enable the plugin for them to work
2. You have ability to manage apps through MAC (as described in MAC) - You can block, deploy to all or deploy to a particular group. However, you cannot control ME separately from a plugin.
3. NA 

I think our messaging is good then :)

**Pull request closed without merging**